### PR TITLE
Add initial styling

### PR DESCRIPTION
**Pull request by jfrankl**

### CODE REVIEW

**Comment by jfrankl** [comment]
Tablet-size:
image

Phone-size:
image

File: src/nih_wayfinding/app/scripts/views/locations/add-location/location-profile/location-profile-controller.js [hunk 1]
```diff
@@ -18,12 +18,12 @@
             ctl.showIconSelect = false;
             ctl.toggleIconSelectDialog = toggleIconSelectDialog;
             ctl.imgOpts = [
-                { img: 'http://lorempixel.com/200/100/' },
-                { img: 'http://lorempixel.com/200/100/' },
-                { img: 'http://lorempixel.com/200/100/' },
-                { img: 'http://lorempixel.com/200/100/' },
-                { img: 'http://lorempixel.com/200/100/' },
-                { img: 'http://lorempixel.com/200/100/' },
+                { img: 'http://localhost:8001/images/icons/icon-house.svg' },
```
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/views/locations/add-location/location-profile/location-profile-controller.js]: Does changing these to `/images/icons/icon-house.svg` not work?

File: src/nih_wayfinding/app/scripts/views/options-grid/options-grid-partial.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <div class="options-grid">
```
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/views/options-grid/options-grid-partial.html]: Looks like we're using the inline template in the actual directive. This html partial should just be deleted.

File: src/nih_wayfinding/app/styles/components/_type.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+h1, h2, h3, h4, h5, h6 {
+  &:first-child {
```
Comment (CloudNiner) [inline src/nih_wayfinding/app/styles/components/_type.scss:2]: Oh. Nice trick.

**Comment by CloudNiner** [comment]
+1 after cleaning up the img urls and the superfluous template partial

File: src/nih_wayfinding/app/styles/components/_type.scss (hunk 3 above)
Comment (jfrankl) [inline src/nih_wayfinding/app/styles/components/_type.scss:2]: I find this really helpful in applications, where you generally just want your headings at the top of the element and handle spacing by adding padding to the element.

**Pull request merged**